cypress/integration/rich-text/RichTextEditor.spec.ts

```diff
@@ -360,6 +360,67 @@ describe('Rich Text Editor', { viewportHeight: 2000 }, () => {
             richText.expectValue(expectedQuoteValue);
           });
         }
+
+        it('should be deleted if empty when pressing delete', () => {
+          richText.editor.click();
+
+          clickDropdownItem(type);
+
+          richText.editor.type('{enter}');
+
+          cy.findByTestId('toolbar-entity-dropdown-toggle').click();
+          cy.findByTestId('toolbar-toggle-embedded-entry-block').click();
+
+          // To make sure paragraph/heading is present
+          richText.expectValue(
+            doc(
+              block(type, {}, text('')),
+              block(BLOCKS.EMBEDDED_ENTRY, {
+                target: {
+                  sys: {
+                    id: 'example-entity-id',
+                    type: 'Link',
+                    linkType: 'Entry',
+                  },
+                },
+              }),
+              block(BLOCKS.PARAGRAPH, {}, text(''))
+            )
+          );
+
+          richText.editor.click('bottom').type('{uparrow}').type('{uparrow}').type('{del}');
```

Comment:
I've tried `.type('{uparrow}{uparrow}')` but I think it triggers it too fast that the browser doesn't catch it sometimes and result in flakiness.